api/config/general.php

```diff
@@ -33,7 +33,7 @@
         'aliases' => [
             '@webroot' => dirname(__DIR__) . '/web',
             '@webBaseUrl' => App::env('WEB_BASE_URL'),
-            '@previewUrlFormat' => App::env('ALIAS_PREVIEW_URL_FORMAT'),
+            '@previewUrlFormat' => App::env('ALIAS_PREVIEW_URL_FORMAT') . '&secret=' . App::env('NEXT_SECRET_TOKEN'),
```

Comment:
Reusable alias for preview targets that includes the next API preview url + the secret token as am extra param